docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Could not create vmnet interface, permission denied or no entitlement
+
```

Comment:
Budhram, the whole document needs to be given a formatting and language cleanup. Currently I have focused on getting the basics right and keeping it consistent.